.github/workflows/common-workflows.yaml

```diff
@@ -0,0 +1,13 @@
+name: Workflow
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: ["**"]
+
+jobs:
+
+  # golang statis analysis checks
```

Comment:
Misspelling on static

Reply:
resolved, please feel free to use the suggestions with the comments to provide a fix